Neos.Neos/Resources/Public/JavaScript/Services/Tools.js

```diff
@@ -0,0 +1,32 @@
+import {
+	isNil,
+	isEmpty,
+	getItemByKeyValue,
+	getCollectionValueByPath,
+	createCollectionByPath,
+} from "../Helper";
+
+const init = () => {
+	if (isNil(window.NeosCMS)) {
+		window.NeosCMS = {};
+	}
+
+	if (isNil(window.Typo3Neos)) {
+		window.Typo3Neos = {};
+	}
+
+	if (isNil(window.NeosCMS.Tools)) {
+		window.NeosCMS.Tools = {
+			isNil,
+			isEmpty,
+			getItemByKeyValue,
+			getCollectionValueByPath,
+			createCollectionByPath,
+		};
+	}
+
+	// deprecated - to be removed in 8.0
+	window.Typo3Neos.Tools = window.NeosCMS.Tools;
```

Comment:
Shouldn't we call them helpers if they are already called helpers?
As they become API, this should be decided now.

Reply:
Can also call them Helpers... thought Tools sounds better. The folder was called helpers since 5.0 ;)